README.md

```diff
@@ -22,16 +22,21 @@ npm install slackbots
 
 ### Methods
 
-- `getChannels` (return: promise) - returns a list of all channels in the team,
-- `getUsers` (return: promise) - returns a list of all users in the team,
-- `getUser` (return: promise) - gets user by name,
-- `getChannel` (return: promise) - gets channel by name,
-- `getChatId` (return: promise) - it returns or opens and returns a direct message channel ID,
-- `postMessage` - posts a message to channel | group | user by ID,
-- `postTo(name, message [, params, callback])` - posts a message to channel | group | user by name,
-- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
-- `postMessageToUser(name, message [, params, callback])` - posts a direct message by user name,
-- `postMessageToGroup(name, message [, params, callback])` - posts a message to private group by name.
+- `getChannels()` (return: promise) - returns a list of all channels in the team,
+- `getGroups()` (return: promise) - returns a list of all groups in the team,
+- `getUsers()` (return: promise) - returns a list of all users in the team,
+- `getChannel(name)` (return: promise) - gets group by name,
+- `getGroup(name)` (return: promise) - gets channel by name,
```

Comment:
sorry to be picky.. I think this description should be swapped with the description for line 28 above.